src/cast.c

```diff
@@ -2609,7 +2826,7 @@ bool isVoidArrayLiteral(Expression *e, Type *other)
  *      false   failed
  */
 
-bool typeMerge(Scope *sc, TOK op, Type **pt, Expression **pe1, Expression **pe2)
+bool typeMerge(Scope *sc, TOK op, Type **pt, Expression **pe1, Expression **pe2, bool ignorealiastthis)
```

Comment:
All uses I saw of `ignorealiasthis` are `!ignorealiasthis`. Consider inverting the meaning of the flag to something like `checkaliasthis` or whatever.